dev/com.ibm.ws.session.cache/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -22,13 +22,16 @@ libraryRef=Shared Library
 libraryRef.desc=Identifies JCache provider files.
 
 uri=JCache configuration URI
-uri.desc=Vendor specific JCache configuration URI, which is passed to the JCache provider when obtaining the CacheManager.
+uri.desc=Vendor-specific JCache configuration URI, which is passed to the JCache provider when the CacheManager is obtained. This setting is ignored when jCacheManagerRef is used.
 
 properties=JCache Configuration Properties
-properties.desc=List of vendor specific JCache configuration properties, which are passed to the JCache provider when obtaining the CacheManager.
+properties.desc=List of vendor-specific JCache configuration properties, which are passed to the JCache provider when the CacheManager is obtained. This setting is ignored when jCacheManagerRef is used.
```

Comment:
jCacheManagerRef **attribute**

Reply:
The messages in this pull request have already been reviewed under https://github.com/OpenLiberty/open-liberty/pull/17684 by @dmuelle. Unfortunately, that PR introduced a build issue and had to be reverted and now I am un-reverting that PR.